python-client/giskard/testing/tests/metamorphic.py

```diff
@@ -61,6 +64,22 @@ def _compare_prediction(results_df, prediction_task, direction, output_sensitivi
         if prediction_task == SupportedModelTypes.CLASSIFICATION:
             passed_idx = results_df.loc[results_df["prediction"] == results_df["perturbed_prediction"]].index.values
 
+        elif prediction_task == SupportedModelTypes.TEXT_GENERATION:
+            try:
+                import evaluate
+            except ImportError as err:
+                raise LLMImportError() from err
+
+            scorer = evaluate.load("bertscore")
```

Comment:
It's better to add `evaluate.load("bertscore")` to the try-except block since it relies on another extra dependency that we add. In case a person already has evaluate (previously installed for example) but not bert-score he'd also get the same `LLMImportError` with  explanations